internal/engine/image_and_cache_builder.go

```diff
@@ -96,7 +96,7 @@ func (icb *imageAndCacheBuilder) Build(ctx context.Context, iTarget model.ImageT
 	case model.CustomBuild:
 		ps.StartPipelineStep(ctx, "Building Dockerfile: [%s]", userFacingRefName)
 		defer ps.EndPipelineStep(ctx)
-		ref, err := icb.custb.Build(ctx, refToBuild, bd.Command)
+		ref, err := icb.custb.Build(ctx, iTarget.ConfigurationRef.Ref, bd.Command)
```

Comment:
shouldn't this be the DeploymentRef? see other places where we do builds, e.g. https://github.com/windmilleng/tilt/blob/master/internal/engine/image_and_cache_builder.go#L37